backend/app/api/endpoints/knowledge.py

```
@@ -1629,140 +1601,709 @@ def _update_kb_summary_after_deletion(kb_id: int, user_id: int, user_name: str):
+        db=db,
+        user_id=current_user.id,
+        status=request_status,
+    )
+
+
+@permission_request_router.get(
+    "/check/{kb_id}", response_model=PermissionRequestCheckResponse
+)
+def check_pending_request(
+    kb_id: int,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.14)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)